spring-boot-samples/README.adoc

```diff
@@ -144,6 +144,9 @@ The following sample applications are provided:
 | link:spring-boot-sample-jta-narayana[spring-boot-sample-jta-narayana]
 | JTA transactions with Narayana
 
+| link:spring-boot-sample-junit5[spring-boot-sample-junit5]
+| Demonstrates JUnit5-based testing
```

Comment:
In general, it is better to refer to _JUnit Jupiter_ when referring to the new programming model in JUnit 5.

I'm not saying you shouldn't mention "JUnit 5", but the focus should not be on JUnit 5 since JUnit 5 is only an umbrella project.